src/sentry/eventstore/models.py

```diff
@@ -681,6 +682,30 @@ def for_group(self, group: Group):
         return GroupEvent.from_event(self, group)
 
 
+@dataclass
+class IssueEvidence:
+    name: str
+    value: str
+    # Feels like we need more attributes to allow us to decide which things to include in verbose and
+    # condensed displays. We'll use all the evidence in issue details and emails, but only important
+    # rows in slack, for example. Some thoughts:
+    # important: bool
+    # include_in: Sequence[str] # List of places this should be included in, like UI, email, slack.
+    # Maybe too configurable? Also doesn't expand well as we add more integration types. Could be
+    # more general like: UI, alerts_email, alerts_messaging, integration_ticketing, etc.
+
+
+@dataclass
+class IssueOccurrence:
+    fingerprint: Sequence[str]
+    issue_title: str
+    resource_id: str
+    evidence_data: Mapping[str, Any]
```

Comment:
This will be used to show grouping information, and can also be used to help render any ui customizations. I'm not sure if this name is a good reflection of that though.

Reply:
I think I'm into the name. I stared at it and thought for a while and all "issues" will have "evidence" of its occurrence, right? The latter half (data) might be a little confusing given that we also have `evidence_display` but I can't really think of anything better. `evidence_ui_data`? That probably isn't better.

Reply:
Maybe `evidence_explain`?

Reply:
I think i'm going to leave these evidence names as is for now. It's not too hard to change them if we figure out something that makes more sense